doc/source/contribution/documentation/introduction.rst

```diff
@@ -6,6 +6,19 @@ Introduction
 This documentation describes about how to write, generate and manage
 Groonga documentation.
 
+How to clone Groonga repository
+-------------------------------
+
+First, please fork Groonga repository on GitHub because
+you need to build Groonga documentation from the latest source code.
+You just access https://github.com/groonga/groonga and press
+`Fork` button. Now you can clone your Groonga repository.
```

Comment:
We need two backquotes in reST. (We need just one or more backquote in Markdown.)

```suggestion
``Fork`` button. Now you can clone your Groonga repository.
```

Reply:
We may want to end with `:` like other sentences.

```suggestion
`Fork` button. Now you can clone your Groonga repository:
```

Reply:
> We need two backquotes in reST.

Thanks, I will check the document of `reST` too.
- https://www.sphinx-doc.org/ja/master/usage/restructuredtext/basics.html#inline-markup

> We may want to end with : like other sentences.

Sure!

Reply:
fix: [b1b844b](https://github.com/groonga/groonga/pull/1712/commits/b1b844ba54ac60243d35dd3ebfa1c130d709fe4b): Fix the markup in reST.

Reply:
fix: [be7057a](https://github.com/groonga/groonga/pull/1712/commits/be7057a8d1b1ab58911dbd90ef2c8a3ca4a0938d) Use `:` for consistency when the next line explains the command.